packages/core/saved-objects/core-saved-objects-api-server-internal/src/mocks/internal_mocks.ts

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type {
+  SavedObjectsPointInTimeFinderClient,
+  ISavedObjectsRepository,
+} from '@kbn/core-saved-objects-api-server';
+import type { SavedObjectsType } from '@kbn/core-saved-objects-server';
+import type { IKibanaMigrator } from '@kbn/core-saved-objects-base-server-internal';
+
+export const createPITClientMock = (): jest.Mocked<SavedObjectsPointInTimeFinderClient> => {
+  return {
+    find: jest.fn(),
+    openPointInTimeForType: jest.fn(),
+    closePointInTime: jest.fn(),
+  };
+};
+
+export const createRepositoryMock = () => {
```

Comment:
not a blocker: I know these are internal mocks and it doesn't matter to external consumers how we organize implementation/test/mocks. However, WDYT about moving each mock to it's own file that has a descriptive file name?

Reply:
Yea, sounds like a good idea.